src/main/webapp/index.html

```diff
@@ -2,11 +2,14 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Nina's Portfolio</title>
+    <title>Capstone</title>
     <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="makeGraph()">
       <p> Hello </p>
+    <div id="chart_div" style="width: 900px; height: 500px;"></div>
+    <div id="chart_div2" style="width: 900px; height: 500px;"></div>
```

Comment:
if these are for different charts, make the div names more descriptive whenever you decide what they're used for